cellfinder_napari/input_container.py

```diff
@@ -1,8 +1,10 @@
+import dataclasses
 from abc import abstractmethod
 from dataclasses import asdict
 from typing import Optional
 
 
+@dataclasses.dataclass
```

Comment:
Slightly shorter and neater.

```suggestion
from abc import abstractmethod
from dataclasses import asdict, dataclass
from typing import Optional


@dataclass
```